README.md

```diff
@@ -20,7 +20,58 @@ spago install machines
 
 ## Quick start
 
-The quick start hasn't been written yet. Contributions are welcome!
+The examples here use `Identity` for simplicity. Usually, you would use
+a different `Monad`, such as `Effect`, `Aff`, `State`, etc.
+
+There are several ways to build machines. One way is to use `do` syntax,
+for example:
+
+```purescript
+machine1 :: MealyT Identity Unit String
+machine1 = do
+  number <- fromArray [10, 20, 30, 40, 50, 0, 60, 70]
+  scaled <-
+      if number == 0
+          then halt
+          else pure $ div number 2
+  pure $ show scaled
+```
+
+This will create a machine `machine1` which goes through the "inputs"
+from the array. It then checks and halts on any zero input, and otherwise
+scales the inputs (by dividing by 2). The result is then transformed intoa string.
+
+The resulting machine can be materialized via `toUnfoldable unit machine1 :: Array String`. The result is `["5","10","15","20","25"]`.
+
+Another way to write the same machine is using machine composition:
```

Comment:
It may also be useful to note here that we're composing `Step`s and briefly note what a `Step` is. Or perhaps it would be better to note that in the prior section. Either way, steps are fundamental to mealy machines and I think we should add a quick sentence about them in the quick start.

Perhaps:

> A `Step f s a` represents a state transition in the machine. When you run a machine you are executing a series of steps. At each step the machine can stop via the `Halt` constructor or emit a value and construct the rest of the machine.

Though feel free to phrase however you see fit.

Reply:
I agree with this! We are composing machines, not steps, but explaining what a step is is quite useful!